db_test.go

```diff
@@ -124,6 +124,27 @@ func signin(s *SurrealDBTestSuite) interface{} {
 	return signin
 }
 
+func (s *SurrealDBTestSuite) TestLive() {
+	s.Run("Live notifications", func() {
+		live, err := s.db.Live("users")
+		s.Require().NoError(err)
+		notifications, er := s.db.LiveNotifications(live.(string))
+		// create a user
+		s.Require().NoError(er)
+		_, e := s.db.Create("users", map[string]interface{}{
+			"username": "johnny",
+			"password": "123",
+		})
+		s.Require().NoError(e)
+		notification := <-notifications
+		res := notification.Result.(map[string]interface{})
```

Comment:
maybe we can do for safety and check `ok`
```suggestion
		res, ok := notification.Result.(map[string]interface{})
```

Reply:
yes yes @phughk suggested it, accidentally pushed the commit lol